document-generator-company-report/src/test/java/uk/gov/companieshouse/document/generator/company/report/mapping/mappers/exemptions/ApiToExemptionsMapperTest.java

```diff
@@ -20,6 +25,9 @@
 @TestInstance(TestInstance.Lifecycle.PER_CLASS)
 public class ApiToExemptionsMapperTest {
 
+    private static LocalDate EXEMPT_FROM = LocalDate.of(2016,01,01);
+    private static String EXEMPTION_TYPE = "exemption type";
```

Comment:
These can be constants: 
`private static final LocalDate EXEMPT_FROM = LocalDate.of(2016,01,01);`
`private static final String EXEMPTION_TYPE = "exemption type";`